tests/clone-flags.c

```diff
@@ -23,11 +23,26 @@
 static const int child_exit_status = 42;
 static pid_t pid;
 
+static void
+retrieve_userns(pid_t pid, char *userns_buf, size_t userns_buflen)
+{
+	char *fname = xasprintf("/proc/%d/ns/user", pid);
```

Comment:
There has to be `free(fname)` in this function to avoid memory leak.

Reply:
fixed locally.